server/dashboards/preconfigured/app-insights-web-app.ts

```diff
@@ -0,0 +1,505 @@
+      breakpoints: { lg: 1200, md: 996, sm: 768, xs: 480, xxs: 0 }
+    }
+  },
+  dataSources: [
+    {
+      id: "timespan",
+      type: "Constant",
+      params: { values: ["24 hours", "1 week", "1 month"], selectedValue: "24 hours" },
+      format: "timespan"
+    },
+    {
+      id: "requests",
+      type: "ApplicationInsights/Query",
+      dependencies: { timespan: "timespan", queryTimespan: "timespan:queryTimespan", granularity: "timespan:granularity" },
+      params: {
+        table: "requests",
+        queries: {
+          duration: {
+            query: ({ granularity }) => {
+              return `
+               where duration > 0 |
+               summarize avg_duration= round(avg(duration) / 1000, 1) by bin(timestamp, ${granularity}) |
+               order by timestamp asc `
+            },
+            format: { type: "timeline", args: { timeField: "timestamp", valueField: "avg_duration" } }
+          },
+          serverrequests: {
+            query: ({ granularity }) => {
+              return `
+                summarize sum = sum(itemCount) by bin(timestamp, ${granularity}), success |
+                order by timestamp asc `
+            },
+            format: {
+              type: "bars",
+              args: { barsField: "timestamp", seriesField: "success", valueField: "sum", threshold: 1 }
+            }
+          }
+        }
+      }
+    },
+    {
+      id: "exceptions",
+      type: "ApplicationInsights/Query",
+      dependencies: { timespan: "timespan", queryTimespan: "timespan:queryTimespan", granularity: "timespan:granularity" },
+      params: {
+        table: "exceptions",
+        queries: {
+          type: {
+            query: ({ granularity }) => {
+              return `summarize count = count() by type `
+            },
+            format: { type: "pie", args: { value: "count", label: "type", maxLength: 20 } }
+          },
+          mapActivity: {
+            query: () => `
+                     extend location=strcat(client_City, ', ', client_CountryOrRegion)
+                    | summarize location_count=count() by location 
+                    | extend popup=strcat('<b>', location, '</b><br />', location_count, ' exceptions')`
+          },
+          count: {
+            query: () =>
+              ` summarize count = count() `,
+            format: {
+              type: "scorecard",
+              args: {
+                countField: "count",
+                thresholds: [{ value: 0, color: "#2196F3", icon: "bug_report", heading: "Exceptions" }]
+              }
+            }
+          },
+        }
+      }
+    },
+    {
+      id: "customEvents",
+      type: "ApplicationInsights/Query",
+      dependencies: { timespan: "timespan", queryTimespan: "timespan:queryTimespan", granularity: "timespan:granularity" },
+      params: {
+        table: "customEvents",
+        queries: {
+          usercount: {
+            query: () =>
+              `
```

Comment:
please be consistent in the way you write `queries` - so for all of them, either use line feed after opening a new string (like here) or use the same line like in `table: "traces"`